tests/Model/ModelManagerTest.php

```diff
@@ -966,6 +966,57 @@ public function testGetUrlsafeIdentifierNull(): void
         $this->assertNull($model->getNormalizedIdentifier(null));
     }
 
+    public function testAddIdentifiersToQueryWithCustomIdMapping(): void
+    {
+        $managerRegistry = $this->createMock(ManagerRegistry::class);
```

Comment:
```suggestion
        $managerRegistry = $this->createStub(ManagerRegistry::class);
```

Reply:
actually `createStub` is just a proxy for `createMock`.

Reply:
Yes, that's the current situation. But in order to make these chages forward compatible, we should use `createStub()` if there isn't any expectation about the class methods.
Please, see https://github.com/sebastianbergmann/phpunit/issues/3120#issue-322494156.

Reply:
Make sense, so I've updated it in last commit